test/integrationTests/signatureHelp.integration.test.ts

```diff
@@ -0,0 +1,93 @@
+/*--------------------------------------------------------------------------------------------- 
+*  Copyright (c) Microsoft Corporation. All rights reserved. 
+*  Licensed under the MIT License. See License.txt in the project root for license information. 
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+import poll from './poll';
+import { should, expect } from 'chai';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+import { omnisharp } from '../../src/omnisharp/extension';
+
+const chai = require('chai');
+chai.use(require('chai-arrays'));
+chai.use(require('chai-fs'));
+
+suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
+    let fileUri: vscode.Uri;
+    suiteSetup(async function () {
+        should();
+
+        let csharpExtension = vscode.extensions.getExtension("ms-vscode.csharp");
+        if (!csharpExtension.isActive) {
+            await csharpExtension.activate();
+        }
+
+        await csharpExtension.exports.initializationFinished;
+
+        let fileName = 'sigHelp.cs';
+        let dir = path.dirname(testAssetWorkspace.projects[0].projectDirectoryPath);
+        let loc = path.join(dir, fileName);
+        fileUri = vscode.Uri.file(loc);
+        await omnisharp.waitForEmptyEventQueue();
+        await vscode.commands.executeCommand("vscode.open", fileUri);
+    });
+
+    
+    test("Returns response with documentation as undefined when method does not have documentation", async function () {
```

Comment:
There are two types of tests we should be thinking about:
* Unit tests, which test the specifics of the C# extension's signature help implementation. These tests should theoretically be able to run outside of VS Code and shouldn't need to send VS Code commands
* Integration tests, which verify that the C# extension's signature help is hooked up in VS Code and validate any nuances there (eg, trigger characters, etc). By definition, these tests run in VS Code.

Today, we can only test features by writing "integration" tests. At some point, we should think about what we'll need to do to write "unit" tests to test these kinds of implementation details. Not blocking your PR on this, but something to consider (cc @TheRealPiotrP ).